server/api/crud/pipelines.py

```diff
@@ -72,7 +72,7 @@ def list_pipelines(
                 response = kfp_client._run_api.list_runs(
                     page_token=page_token,
                     page_size=mlrun.common.schemas.PipelinesPagination.max_page_size,
-                    sort_by=sort_by,
+                    sort_by=sort_by if page_token == "" else "",
```

Comment:
just a suggestion -
```suggestion
                   # omit the sort_by as page token is provided. kf stores the query parameters when page token is provided
                    sort_by=sort_by if not page_token else "",
```

Reply:
There's already a comment about omitting `filter` above the function call, adding `sort_by` to it as well.